sample/ios-app/tensorflow-test/ViewController.swift

```diff
@@ -15,10 +15,10 @@ class ViewController: UIViewController, SketchViewDelegate {
     private var tfDigitClassifier: TFDigitClassifier?
     
     private var isInterpreterInited: Bool = false
-    
+    private var scope: Kotlinx_coroutines_coreCoroutineScope? = nil
     override func viewDidLoad() {
         super.viewDidLoad()
-    
+        scope = createCoroutineScope()
```

Comment:
this code can't compile with current common code version. and on CI it was failed.
![image](https://user-images.githubusercontent.com/5010169/124493856-4d0c5a00-dde0-11eb-81e8-cef820181aba.png)
we should fix CI - CI should fail on errors. https://github.com/xcpretty/xcpretty/issues/69 please fix here and in all moko libs